src/Components/Form.js

```diff
@@ -0,0 +1,50 @@
+import { useState } from 'react';
+import { IconContext } from 'react-icons';
+import {AiFillPlusCircle}from 'react-icons/ai'
+import HandleUserChoice from './HandleUserChoice';
+const Form = ({addListToday,addListTomorrow,addListUpcoming}) => {
+    const [inputText,setInputText]=useState('');
+    const [showComponent, setShowComponent] = useState(false);
```

Comment:
Which specific component should be displayed? Could you suggest a more descriptive name for this state?